.prettierignore

```diff
@@ -18,6 +18,7 @@ npm-debug.log*
 .github/PULL_REQUEST_TEMPLATE.md
 
 /website/playground
+/website/build
```

Comment:
I guess this was a little unrelated, but got caught running the lint locally after I had tried to build the site separately